assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-footer-block/index.tsx

```diff
@@ -22,7 +24,47 @@ registerBlockType( 'woocommerce/mini-cart-footer-block', {
 			/>
 		),
 	},
-	attributes,
+	deprecated: [
+		{
+			attributes: deprecatedAttributes,
+
+			migrate( attributes, innerBlocks ) {
+				const {
+					cartButtonLabel,
+					checkoutButtonLabel,
+					...restAttributes
+				} = attributes;
+
+				return [
+					restAttributes,
+					[
+						createBlock(
+							'woocommerce/mini-cart-cart-button-block',
+							{
+								cartButtonLabel,
+							}
+						),
+						createBlock(
+							'woocommerce/mini-cart-checkout-button-block',
+							{
+								checkoutButtonLabel,
+							}
+						),
+						...innerBlocks,
+					],
+				];
+			},
+			isEligible: ( attributes ) => {
+				return (
+					objectHasProp( attributes, 'cartButtonLabel' ) &&
+					objectHasProp( attributes, 'checkoutButtonLabel' )
```

Comment:
Should this be an `or` instead of an `and`? Otherwise if only one label was customized, it's not considered for the deprecation.
```suggestion
					objectHasProp( attributes, 'cartButtonLabel' ) ||
					objectHasProp( attributes, 'checkoutButtonLabel' )
```

Reply:
You are right! 👍